<!DOCTYPE html>
<html>
<body>
<p>&nbsp;</p> 
<h4><i>Data strongly suggests that:<br>
<h5>- GC skew values can be used to identify genes that are in an inverted state relative to their typical orientation
<br>
<br>
- GC skew values based exclusively on codon-position 1 nucleotides may be the most reliable indicator of a gene's typical orientation, and therefore if it is in the typical or inverted state.
</h5>
<br>
<h4>Furthermore:</h4>
<h5>- GC skew data suggest that the replication fork is a stronger determinant of a gene's GC skew value than selection.</i></h5>
<br>
<h4>Discussion</h4>
<p>Comparison of the CP1-based versus CP3-based GC skew values for the same genes shows that CP3-based GC skew values are generally higher than the CP1-based values for the same genes. Addtionally, CP3-based values are largely positive, even in genes that have an overall whole-gene-based negative GC skew. Also, CP3-based values are largely indistinguishable between genes on the leading and lagging strands. Together these <b>data strongly favor hypothesis 1</b>, that mutational pressure drives increasingly positive GC skew values in all genes, irrespective of their orientation. Notably, as these data revear the net effect of multiple sources of selection and mutagenesis, the data do not contradict the model that selection drives GC skew values to increse with respect to a gene's coding strand (which was proposed to drive increasingly negative GC skew values in lagging strand genes). Rather, the data seem to suggest that the effects of replication generally overwhelm the effects of selection.</p>

<p>As replication-based mutagenesis appears to be the most influential driver of GC skew values, <b>negative GC skew values should  indicate that a gene is in its atyical orientation</b>. Therefore, it should be possible to identify genes at the whole-genome level currently (in a given genome) encoded in the atpical orientation as per our original study.</p>

<p>Notably, our comparison of codon-position based GC skews also suggests that <b>CP1-based GC skew values should yield higher fidelity data</b> than our original whole-gene method. This is because negative selection should prevent the fixation of CP1 mutations, thereby preventing negative GC skew values from increasing after a flip occurs. This is supported by additional (as yet unpublished) analyses we performed in a separate study. There we compared phylogeny-based gene inversion detection with GC skew based detection and identified 100% agreement between the two method when using the CP1-based, but not the CP3-based GC skew calculations.</p>

<p>Overall, these <b>data support the validity of our original conclusions</b>. In fact, the CP1-based analyses also shows that nearly every lagging strand gene has a negative CP1-based GC skew, implying that nearly every lagging strand gene was originally encoded on the leading strand. If correct, this observation reveals an overwhelming trend across all bacterial species, providing significant insight into the function and evolutionary history of bacterial genome architectures.</p>